galaxy_ng/app/api/urls.py

```diff
@@ -1,18 +1,26 @@
 from django.conf import settings
 from django.urls import include, path
 
+from rest_framework import routers
+
 from . import views
+from .v3 import viewsets
```

Comment:
Abstractions leak.
`galaxy_ng.app.api.urls` must not refer to v3 viewsets directly.

Reply:
Not sure I follow.  Are you suggesting moving the namespace router setup to api/v3/urls.py?

Reply:
moved the router setup to api/v3/urls.py